src/css/components/_error.scss

```diff
@@ -1,48 +1,4 @@
-.vjs-error-display {
-  display: none;
-}
-
-.vjs-error .vjs-error-display {
-  display: block;
-  position: absolute;
-  left: 0;
-  top: 0;
-  width: 100%;
-  height: 100%;
-}
-
-.vjs-error .vjs-error-display:before {
-  content: 'X';
```

Comment:
Yeah. It looked totally broken with that style - it duplicates a lot of the style of the modal dialog.

I don't know what the "X" was about (I was guessing a "close" icon/button).